config/core/resources/broker.yaml

```diff
@@ -154,6 +154,10 @@ spec:
     - name: Reason
       type: string
       jsonPath: ".status.conditions[?(@.type==\"Ready\")].reason"
+    - name: Class
```

Comment:
the name is ok... or different?

Reply:
In the example in the issue, the name is supposed to be `CLASS` so that's why i wrote thiss

Reply:
OK :-)